### PR TITLE
fix(update):fix update bugs

### PR DESCRIPTION
**Pull request by ni00**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/main/.github/CONTRIBUTING.md)

#### What is the current behavior?
1. If the new version is not installed, just click Install and cancel, the next startup will still display the update information window。
2. The dark mode style is not correct
3. Components are not set to load asynchronously, which may affect startup speed
4. The dialog is too small

#### Issue Number

None

#### What is the new behavior?
- [x] 1. Record the version number to pop up the update log window
- [x] 2. Set dark mode style 
- [x] 3. Setting asynchronous components
- [x] 4. modify dialog style

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Specific Instructions

Are there any specific instructions or things that should be known prior to review?

### CODE REVIEW

**Review by Red-Asuka — APPROVED** [review]

File: src/views/update/index.vue [hunk 1]
```diff
@@ -64,17 +64,55 @@ export default class Update extends Vue {
   @Getter('currentTheme') private theme!: Theme
   @Getter('currentLang') private getterLang!: Language
   @Getter('autoCheck') private autoCheck!: boolean
+  @Ref('detail_display') private detail_display!:HTMLDivElement
```
Comment (ysfscream) [inline src/views/update/index.vue:67]:
```suggestion
  @Ref('detail_display') private detail_display!: HTMLDivElement 
```

File: src/views/update/index.vue [hunk 2]
```diff
@@ -64,17 +64,55 @@ export default class Update extends Vue {
...
   @Getter('autoCheck') private autoCheck!: boolean
+  @Ref('detail_display') private detail_display!:HTMLDivElement 
   private showDialog: boolean = false
   private progressVisible: boolean = false
-  private updateDetail: versionDetail = {
-    version: '',
-    detail: '',
-  }
+  private version:string = ''
+  private detail:string = ''
```
Comment (ysfscream) [inline src/views/update/index.vue:71]:
```suggestion
  private version: string = ''
  private detail: string = ''
```

File: src/views/update/index.vue [hunk 3]
```diff
@@ -64,17 +64,55 @@ export default class Update extends Vue {
...
+        for(let a of aTags){
+          a.onclick =  (e) =>{
+            e.preventDefault()
+            this.goToLink(a.href)
+            return false;
+        }
+        }
+      }
+    )
+  }
```
Comment (ysfscream) [inline src/views/update/index.vue:98]:
```js
function handleATags() {
  this.$nextTick(() => {
    const aTags = this.detail_display.getElementsByTagName('a')
    for (let a of aTags) {
      a.onclick = (e) => {
        e.preventDefault()
        this.goToLink(a.href)
        return false
      }
    }
  })
}
```

Please pay attention to the code formatting standards.

File: src/views/update/index.vue [hunk 4]
```diff
@@ -64,17 +64,55 @@ export default class Update extends Vue {
...
+    const def_w = 900
+    if (w < def_w) {
+      this.dialogWidth = '100%'
+    } else {
+      this.dialogWidth = def_w + 'px' 
+    }
+
+    this.dialogHeight = String(Math.floor(h*0.5)) + 'px' 
+    this.contentHeight = String(Math.floor(h*0.5) - 180) + 'px' 
+  }
```
Comment (ysfscream) [inline src/views/update/index.vue:112]: Compared to using variables like w and h, width and height make the code more readable.

File: src/views/update/index.vue [hunk 5]
```diff
@@ -64,17 +64,55 @@ export default class Update extends Vue {
...
-    version: '',
-    detail: '',
-  }
+  private version:string = ''
+  private detail:string = ''
   private progress: number = 0
   private downloaded: boolean = false
+  private dialogWidth:string = '900px'
+  private dialogHeight:string = '350px'
+  private contentHeight:string = '170px'
```
Comment (ysfscream) [inline src/views/update/index.vue:76]:
```suggestion
  private dialogWidth: string = '900px'
  private dialogHeight: string = '350px'
  private contentHeight: string = '170px'
```

**Review by ysfscream — APPROVED** [review]

**Pull request merged**